clang/test/CodeGenHLSL/GlobalConstructorFunction.hlsl

```diff
@@ -25,7 +25,7 @@ void main(unsigned GI : SV_GroupIndex) {}
 //CHECK-NEXT: entry:
 //CHECK-NEXT:   call void @"?call_me_first@@YAXXZ"()
 //CHECK-NEXT:   call void @"?then_call_me@@YAXXZ"()
-//CHECK-NEXT:   %0 = call i32 @llvm.dx.flattened.thread.id.in.group()
+//CHECK-NEXT:   %0 = call i32 @llvm.[[ICF:dx]].flattened.thread.id.in.group()
```

Comment:
We can skip tests like this that don't yet have a SPIRV run line. That is, if there's no `-triple spirv-*` in a `RUN:` line at the top, you can skip the test for now.

Reply:
Thank you done